crates/libp2p-networking/src/network/behaviours/dht/mod.rs

```diff
@@ -330,7 +330,7 @@ impl<K: SignatureKey + 'static> DHTBehaviour<K> {
                     // Only return the record if we can store it (validation passed)
                     if store.put(record).is_ok() {
                         if notify.send(r).is_err() {
-                            error!("Get DHT: channel closed before get record request result could be sent");
+                            warn!("Get DHT: channel closed before get record request result could be sent");
```

Comment:
Same comment this probably is a real error if the chan is closed although this one I could see happening if the store query is just slow and whoever sent it timedout and dropped the rx side of the chan

Reply:
Yeah same as above, we see it when we insta-cancel our own requests if we already have an outbound one for the same key